dht/compositions/find-provs-both-local.toml

```diff
@@ -5,14 +5,13 @@
 [global]
   plan = "dht"
   case = "find-providers"
-  total_instances = 150
   builder = "docker:go"
   runner = "local:docker"
 
 [[groups]]
   id = "balsam-undialable-provider"
   [groups.instances]
-    count = 5
+    count = 16
```

Comment:
Why did you changed this numbers? I'm not attached to these values, but is it required to make the test pass?